src/api/status/src/server.js

```diff
@@ -1,6 +1,7 @@
 const { Satellite } = require('@senecacdot/satellite');
 const { static } = require('express');
 const path = require('path');
+const host = process.env.API_HOST || 'localhost';
```

Comment:
Let's do a few things here.  `API_HOST` should be defined.  Let's add it to https://github.com/Seneca-CDOT/telescope/blob/master/src/api/status/env.local as `API_HOST=localhost`.

Also, can you fix https://github.com/Seneca-CDOT/telescope/blob/master/config/env.development#L21 to drop the `http://` so that we're consistently using a host vs. origin everywhere.

Reply:
So by adding/changing those two things, we won't need this line anymore?

Reply:
We can do all 3 (leave this line and also change the other 2)